src/utils/regexwrappers.cpp

```diff
@@ -1,11 +1,16 @@
 #include "regexwrappers.hpp"
 #include "regexes.hpp"
 #include "utils.hpp"
+#include <charconv>
 #include <fmt/core.h>
 
 inline int svtoi(const std::string_view sv)
 {
-    return std::atoi(sv.data());
+    int value{};
+    auto [ptr, ec] = std::from_chars(sv.begin(), sv.end(), value);
+    if (ec != std::errc() || ptr != sv.end())
```

Comment:
Actually you can’t use .begin(), .end() as it’s not guaranteed it will be a pointer type, only some standard libraries implements it as pointer.

Reply:
TIL! Makes sense though - thanks hana. This code is soon for the cutting-room floor in favour of your to_number :)

Reply:
if you know that the data are contiguous you can use `sv.data(), sv.data() + sv.size()` there is a paper in pipeline to allow from_chars for string_views…

Reply:
Ha, from the signature I would have assumed from_chars would be like other std functions like accumulate, foreach etc and accept iterators, but nope